test/optimization/test_qaoa.py

```diff
@@ -84,7 +83,3 @@ def test_qaoa(self, w, prob, m, solutions):
         self.log.debug('solution:           %s', graph_solution)
         self.log.debug('solution objective: %s', max_cut.max_cut_value(x, w))
         self.assertIn(''.join([str(int(i)) for i in graph_solution]), solutions)
-
```

Comment:
Why are you removing this - it facilitates things for when running it as a single test.